Sources/Kevin/Accounts/BankSelection/Cells/BankHeaderCell.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  BankHeaderCell.swift
+//  
+//
+//  Created by Arthur Alehna on 01/09/2022.
+//
```

Comment:
Probably need to update comment to include copyright as it is done in KevinBAnkSelectionView.swift